addons/project/tests/test_analytic_account_consistency.py

```diff
@@ -0,0 +1,61 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+from odoo.tests import Form, tagged
+from odoo.exceptions import UserError
+
+@tagged('-at_install', 'post_install')
+class TestAnalyticAccountConsistency(TestProjectCommon):
+
+    def test_account_consistency_orm_method(self):
+        """
+        This test ensures that when the user validates his changes in the list view of analytic account, the account
+        are correctly removed from their project.
+        """
+
+        project_account_id, project_multiple_account = self.env['project.project'].create([{
+            'name': 'project_account_id',
+        }, {
+            'name': 'project_multiple_account',
+        }])
+        analytic_plan = self.env['account.analytic.plan'].create({'name': 'secondary plan'})
+        project_plan = self.env['account.analytic.plan'].browse(int(self.env['ir.config_parameter'].get_param('analytic.project_plan', 0)))
```

Comment:
To avoid creating too much columns in the DB, let's reuse the already defined plans and set them in the `setUpClass()`
```suggestion
        cls.project_plan, cls.other_plans = cls.env['account.analytic.plan']._get_all_plans()
```

Reply:
And only if there is nothing in `cls.project_plan` and/or `cls.other_plans`, then let's create new plans